CHANGELOG.md

```diff
@@ -23,7 +23,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 - [#2936](https://github.com/thanos-io/thanos/pull/2936) Compact: Fix ReplicaLabelRemover panic when replicaLabels are not specified.
 
 ### Added
-
+- [#2849](https://github.com/thanos-io/thanos/pull/2849) Request Logging : Added request logging for HTTP server side.
```

Comment:
```suggestion
- [#2849](https://github.com/thanos-io/thanos/pull/2849) <mention components affected>: Added request logging for HTTP server side.
```

Reply:
Let's focus also on what it means for the users. How to use it?

Reply:
Yeah, I need to discuss how do we want to enable the user to use this feature

Reply:
Consensus - https://cloud-native.slack.com/archives/CL25937SP/p1596518076006500